doc/sphinx/addendum/micromega.rst

```diff
@@ -112,11 +116,11 @@ and checked to be :math:`-1`.
 .. tacn:: lia
    :name: lia
 
-This tactic offers an alternative to the :tacn:`omega` and :tacn:`romega`
-tactics. Roughly speaking, the deductive power of lia is the combined deductive
-power of :tacn:`ring_simplify` and :tacn:`omega`. However, it solves linear
-goals that :tacn:`omega` and :tacn:`romega` do not solve, such as the following
-so-called *omega nightmare* :cite:`TheOmegaPaper`.
+   This tactic offers an alternative to the :tacn:`omega` and :tacn:`romega`
```

Comment:
Just a heads-up, #8419 removes `romega` and will be merged soon and it will create a conflict here.